docs/controls/WrapPanel.md

```diff
@@ -44,15 +42,14 @@ When the Orientation is Vertical, HorizontalSpacing adds uniform spacing between
 The following sample demonstrates how to add WrapPanel Control.
 
 ```xaml
-
 <Page x:Class="Microsoft.Toolkit.Uwp.SampleApp.SamplePages.WrapPanelPage"
-      xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
-      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
-      xmlns:local="using:Microsoft.Toolkit.Uwp.SampleApp.SamplePages"
-      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
-      xmlns:wrapPanel="using:Microsoft.Toolkit.Uwp.UI.Controls.WrapPanel"
-      mc:Ignorable="d">
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
Lots of whitespace changes, these don't lineup anymore. :(

Reply:
It was Visual Studio, It is probably better because that means these will take less horizontal space.

Reply:
I just like my xaml attributes to line up, but maybe I'm weird that way...

Reply:
It would a major pain to fix all of the code comments :(